content/client-server-api/modules/event_replacements.md

```diff
@@ -231,15 +244,14 @@ subsequent replacements are themselves redacted). Note that this behaviour is
 specific to the `m.replace` relationship. See also [redactions of edited
 events](#redactions-of-edited-events) below.
 
-##### Server-side replacement of content
-
-Whenever an `m.replace` is to be bundled with an event as above, the server
-should also modify the content of the original event according to the
-`m.new_content` of the most recent replacement event (determined as above).
+{{< changed-in v="1.7" >}} In previous versions of the specification, only the
+`event_id`, `origin_server_ts`, and `sender` of the replacement event were included
+in the aggregation. v1.7 extends the aggregation to the full event.
 
-An exception applies to [`GET /_matrix/client/v3/rooms/{roomId}/event/{eventId}`](#get_matrixclientv3roomsroomideventeventid),
-which should return the unmodified event (though the relationship should still
-be bundled, as described above).
+{{< changed-in v="1.7" >}} In previous versions of the specification, servers
+were expected to replace the content of the original event with that of the
+replacement event. However that behaviour made reliable client-side
+implementation difficult, and servers should no longer make this replacement.
```

Comment:
As far as spec language is concerned, this should be something like:

```
{{< changed-in v="1.7" >}} Servers MUST NOT return the most recent edit from [/event].

{{ boxes/rationale }}
In previous versions...
{{ /boxes/rationale }}
```

Reply:
> ```
> {{< changed-in v="1.7" >}} Servers MUST NOT return the most recent edit from [/event].
> ```

this seems to say the opposite to what is intended. 

I've moved the `changed-in` to the top of the section, which seems to be what we've done elsewhere, and moved this explanation into a "rationale" as you suggest.